### PR TITLE
SDSS/DSS Finder charts

### PR DESCRIPTION
**Pull request by profxj**
And a few other bits.  Here are the CHANGES:

- Convenience function for a list of hosts
- SDSS/DSS finder charts

### CODE REVIEW

File: frb/figures/finder.py [hunk 1]
```diff
@@ -5,16 +5,43 @@
...
+def from_hdu(hdu, title, **kwargs):
+    """
+    Convenience function to handle an HDU to generate a finder chart
+
+    Args:
+        hdul (astropy.io.fits.PrimaryHDU):
+        title (str):
+
+    Returns:
+        see generate()
```
Comment (ntejos) [inline frb/figures/finder.py]: Can you copy the documentation of generate() here as well? At least the object returned, with a brief explanation: matplotlib.pyplot.figure, matplotlib.pyplot.Axis

File: frb/surveys/sdss.py [hunk 2]
```diff
@@ -131,18 +131,21 @@ def get_catalog(self, photoobj_fields=None, timeout=120, print_query=False):
...
-        trim_catalog['z_spec'] = zs
+        if spec_catalog is not None:
+            trim_spec_catalog = trim_down_catalog(spec_catalog)
+            # Match
+            spec_coords = SkyCoord(ra=trim_spec_catalog['ra'], dec=trim_spec_catalog['dec'], unit='deg')
+            phot_coords = SkyCoord(ra=trim_catalog['ra'], dec=trim_catalog['dec'], unit='deg')
+            idx, d2d, d3d = match_coordinates_sky(spec_coords, phot_coords, nthneighbor=1)
+            # Check
+            if np.max(d2d).to('arcsec').value > 1.5:
+                embed(header='119 of sdss')
```
Comment (ntejos) [inline frb/surveys/sdss.py]: what does '119 of sdss' mean?'

File: frb/figures/finder.py (hunk 1 above)
Comment (profxj) [reply frb/figures/finder.py]: done

File: frb/surveys/sdss.py (hunk 2 above)
Comment (profxj) [reply frb/surveys/sdss.py]: replaced with an Exception.

**Pull request merged**